js/README.md

```diff
@@ -67,7 +70,7 @@ import { hostCapabilities } from 'kubewarden-policy-sdk';
 
 // Get OCI manifest
 const manifest = hostCapabilities.OciManifest.getManifest('registry.io/image:tag');
-console.log('Manifest:', manifest);
+console.error('Manifest:', manifest);
```

Comment:
This is fine, we [emit warnings](https://github.com/kubewarden/policy-evaluator/blob/main/src/runtimes/wasi_cli/runtime.rs#L40-L47) for these